sdk/cosmos/azure-spring-data-cosmos/src/test/java/com/azure/spring/data/cosmos/core/converter/MappingCosmosConverterUnitTest.java

```diff
@@ -93,12 +94,10 @@ public void canReadPojoWithDateFromDocument() throws ParseException {
         jsonObject.put(TestConstants.PROPERTY_DATE, date);
         jsonObject.put(TestConstants.PROPERTY_ID, TestConstants.ID_1);
 
-        final JsonNode jsonNode = mappingCosmosConverter.writeJsonNode(jsonObject);
```

Comment:
same issue as described above.

Also, the assertion in this test just seemed wrong and it was comparing jsonNode contents instead of the fields on the memo object

Reply:
Thanks for fixing this.